datadog/resource_datadog_synthetics_test_.go

```diff
@@ -684,35 +751,37 @@ func updateSyntheticsTestLocalState(d *schema.ResourceData, syntheticsTest *data
 	d.Set("locations", syntheticsTest.Locations)
 
 	actualOptions := syntheticsTest.GetOptions()
-	localOptions := make(map[string]string)
+	localOptions := make([]map[string]string, 1)
+	localOption := make(map[string]string)
 	if actualOptions.HasFollowRedirects() {
-		localOptions["follow_redirects"] = convertToString(actualOptions.GetFollowRedirects())
+		localOption["follow_redirects"] = convertToString(actualOptions.GetFollowRedirects())
 	}
 	if actualOptions.HasMinFailureDuration() {
-		localOptions["min_failure_duration"] = convertToString(actualOptions.GetMinFailureDuration())
+		localOption["min_failure_duration"] = convertToString(actualOptions.GetMinFailureDuration())
 	}
 	if actualOptions.HasMinLocationFailed() {
-		localOptions["min_location_failed"] = convertToString(actualOptions.GetMinLocationFailed())
+		localOption["min_location_failed"] = convertToString(actualOptions.GetMinLocationFailed())
 	}
 	if actualOptions.HasTickEvery() {
-		localOptions["tick_every"] = convertToString(actualOptions.GetTickEvery())
+		localOption["tick_every"] = convertToString(actualOptions.GetTickEvery())
 	}
 	if actualOptions.HasAcceptSelfSigned() {
-		localOptions["accept_self_signed"] = convertToString(actualOptions.GetAcceptSelfSigned())
+		localOption["accept_self_signed"] = convertToString(actualOptions.GetAcceptSelfSigned())
 	}
 	if actualOptions.HasAllowInsecure() {
-		localOptions["allow_insecure"] = convertToString(actualOptions.GetAllowInsecure())
+		localOption["allow_insecure"] = convertToString(actualOptions.GetAllowInsecure())
 	}
 	if actualOptions.HasRetry() {
 		retry := actualOptions.GetRetry()
-		localOptions["retry_count"] = convertToString(retry.GetCount())
+		localOption["retry_count"] = convertToString(retry.GetCount())
 
 		if interval, ok := retry.GetIntervalOk(); ok {
-			localOptions["retry_interval"] = convertToString(interval)
+			localOption["retry_interval"] = convertToString(interval)
 		}
 	}
-
-	d.Set("options", localOptions)
+	localOptions = append(localOptions, localOption)
+	d.Set("options", localOption)
+	d.Set("options_list", localOptions)
```

Comment:
I don't think you can/should set both. See what we did with assertions, I think it's roughly the same.